.npmignore

```diff
@@ -1,2 +1,5 @@
 test/*
-
+.editorconfig
+.eslintrc
```

Comment:
`npm run lint` won't work for users of this package because `jscs` and `eslint` are developer dependencies - not usage dependencies.